delivery.yaml

```diff
@@ -8,22 +8,25 @@ pipeline:
           apt-get install -y \
           openjdk-8-* \
           maven \
-          git
+          git \
+          python3.5 \
+          python3-pip 
           curl -fLOsS https://delivery.cloud.zalando.com/utils/ensure-docker && sh ensure-docker && rm ensure-docker
 
       - desc: "Build maven package"
         cmd: |
           ./mvnw clean package
 
+          # Our Dockerfiles still need to be compatible with Jenkins deployments and local builds:
+          pip3 install --upgrade scm-source
+          scm-source -f target/scm-source.json
+
       - desc: "Push Docker Image"
         cmd: |
-          IS_PR_BUILD=${CDP_PULL_REQUEST_NUMBER+"true"}
-          if [[ ${IS_PR_BUILD} != "true" ]]
-          then
-            RELEASE_VERSION=$(git describe --tags --always --dirty)
-            AGENT_IMAGE=registry-write.opensource.zalan.do/zmon/zmon-data-service:${RELEASE_VERSION}
-          else
+          if [ -z "$CDP_PULL_REQUEST_NUMBER" ]; then
             AGENT_IMAGE=registry-write.opensource.zalan.do/zmon/zmon-data-service:${CDP_BUILD_VERSION}
+          else
+            AGENT_IMAGE=pierone.stups.zalan.do/zmon/zmon-data-service:${CDP_BUILD_VERSION}
```

Comment:
I'm going to be grumpy here and ask: is it OK to expose `pierone.stups.zalan.do` here?
Probably we should just close-source ZMON -___-

Reply:
I would think that is fine as it is in all the CDP examples.  :)